SWLOR.Game.Server/Feature/AbilityDefinition/ThrowLightsaberAbilityDefinition.cs

```diff
@@ -116,6 +116,18 @@ private static void ThrowLightsaber1(AbilityBuilder builder)
                 .RequirementFP(3)
                 .IsCastedAbility()
                 .DisplaysVisualEffectWhenActivating()
+                .HasCustomValidation((activator, target, level) =>
+                {
+                    var weapon = GetItemInSlot(InventorySlot.RightHand);
```

Comment:
Might be a good idea to throw this logic into a function and reuse it across each level. Less maintenance overhead if it's in one place.